src/test/java/io/lighty/yang/validator/IntegrationTest.java

```diff
@@ -0,0 +1,62 @@
+package io.lighty.yang.validator;
```

Comment:
Add license header (to all files where it is missing)

Reply:
done